### DISCUSSION TITLE
Projects Beta: Show the linked PR in the Issue slide over preview

### DISCUSSION THREAD

**Discussion by drbilo** [community/community#37834] [Projects and Issues]
When clicking on an issue from the projects beta board, it would be great to be able to see any linked PRs in the issue slideover/preview pane that comes from the right hand side. For some reason they don't show in this view and I have to click into the actual issue to see any linked PRs. It would make life much easier if I could see these in the slideover

**Comment by ossanna16** [comment]
Hi there @drbilo and welcome to our community! Thank you for the feedback!

**Comment by alexanderniebuhr** [comment]
Hey @drbilo would be great if you can show an screenshot as an example. I can see the linked PR in the _right-hand slide_

<img width="1501" alt="Screenshot 2022-11-02 at 21 08 25" src="https://user-images.githubusercontent.com/45965090/199592951-a835458f-78f1-4828-ab4c-b60345be86dc.png">

**Comment by drbilo** [comment]
Hey @alexanderniebuhr. Here's a short video of what I'm seeing:


https://user-images.githubusercontent.com/24389345/200514255-9e47a087-db8d-4041-995a-02144111a39c.mp4

It could be because we use `ref` for linking PRs rather than `close`? We use this workflow here so it would be great if these could show up here too

**Reply by alexanderniebuhr** [reply]
@drbilo thanks. Does the PR show up in the normal issue view on the right side as a linked PR in the **Development** Section. If not, that might be the issue.

<img width="320" alt="Screenshot 2022-11-08 at 12 23 48" src="https://user-images.githubusercontent.com/45965090/200552013-78cb4297-9c2c-4145-b93e-13f74aa05a9f.png">

**Reply by drbilo** [reply]
> @drbilo thanks. Does the PR show up in the normal issue view on the right side as a linked PR in the **Development** Section. If not, that might be the issue.
> 
> <img alt="Screenshot 2022-11-08 at 12 23 48" width="320" src="https://user-images.githubusercontent.com/45965090/200552013-78cb4297-9c2c-4145-b93e-13f74aa05a9f.png">

Yeah I think that's it. As we use Ref # because we don't want to close the issue until it has reached our production env. Would be nice if Ref # would show up here then
